e2e/entrypointl2.sh

```diff
@@ -36,7 +36,10 @@ sh /tmp/genesisl2.sh
  /tmp/datadir \
  --authrpc.vhosts="*" \
  --authrpc.addr=0.0.0.0 \
- --authrpc.port=8551 \
+--authrpc.port=8551 \
```

Comment:
I think the indentation of this line was changed 🤔

Reply:
ah nope will fix